home.html

```diff
@@ -264,7 +264,7 @@ <h2 class = "text-center"><strong>Annual Events:</strong></h2>
                                 <h5 class="card-title"><strong>DevSoc and Dragons</strong></h5>
                                 <p class="card-text">Exchanging their keyboards and mice for swords and magic, DevSoc enters the world of fantasy.</p>
                                 <p class="card-text text-right"><i>Annual Summer Event</i></p>
-                                <a href="Dnd/Dnd.html" target = "_blank" class="btn btn-dark float-right">View content</a>

```

Comment:
Is it intentional that this no longer opens in a new tab/window?
(removing `target="_blank"` will make it open in the current tab instead of a new one)

Reply:
Yeah that's intentional, is that okay or would you like me to change it? I just thought it looked nicer and was more consistent with how the archive page works right now.

Reply:
Nah, that's fine - I think the user experience might be nicer if you open in a new tab, but if it's consistent with what's already there, then I don't think it matters that much really.
Probably something to review another time instead.